database/migrations/2024_04_09_190804_create_bootcamps_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('bootcamps', function (Blueprint $table) {
+            $table->uuid('id')->primary();
+            $table->string('name');
+            $table->timestamps();
```

Comment:
No, they are not necessary and can be deleted.

Reply:
We have been discussing with @FranEnLaNube and maybe it could be necessary. Not resolved yet.

Reply:
We can delete it, but, as we don't know yet how to implement the event bus, in order to finish this task, we prefer to leave it for a next task.
And, in case to do it, could be necessary to do it at all the tables, exceding the scope of this task